src/components/KanbanBoard/KanbanBoard.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import KanbanColumn from '../KanbanColumn/KanbanColumn';
+
+class KanbanBoard extends Component {
+  constructor(props) {
+    super(props)
+  }
+  render() {
+    const cards = this.props.cards;
+
+
+    const inQueue = cards.filter((card) => {
```

Comment:
You want to move the filtering to the KanbanColumn component rather than do it all upfront over here.